src/kv/serialise_entry_json.h

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+#pragma once
+
+#include <nlohmann/json.hpp>
+#include <vector>
+
+namespace kv::serialisers
+{
+  template <typename T>
+  struct JsonSerialiser
+  {
+    using Bytes = std::vector<uint8_t>;
```

Comment:
If we're going to have Bytes, we should probably have it somewhere common to at least the serialisers.